ote/src/clj/ote/tasks/pre_notices.sql

```diff
@@ -16,6 +16,34 @@ SELECT id, "pre-notice-type", "route-description", created, modified, descriptio
    AND ((:regions::char(2)[]) IS NULL OR (:regions::char(2)[]) && n.regions)
    AND (sent > (current_timestamp - :interval::interval));
 
+-- name: fetch-unsent-changes-by-regions
```

Comment:
Just double-checking, is cross-join the best approach here instead of e.g. (inner) JOIN...ON?